crates/sui-core/src/state_accumulator.rs

```diff
@@ -60,6 +79,25 @@ impl StateAccumulator {
                 .collect::<Vec<ObjectDigest>>(),
         );
 
+        // insert wrapped tombstones. We use a custom struct in order to contain the tombstone
+        // against the object id and sequence number, as the tombstone by itself is not unique.
+        acc.insert_all(
+            effects
+                .iter()
+                .flat_map(|fx| {
+                    fx.wrapped()
+                        .iter()
+                        .map(|oref| {
+                            serde_json::to_string(&WrappedObject::new(oref.0, oref.1))
```

Comment:
we need to use BCS, not json. you should be able to use write `bcs::to_bytes` - there are plenty of examples of this in the code base if you have any trouble